### PR TITLE
Tạo Frontend Layout

### PR DESCRIPTION
**Pull request by haunc-0912**
==> Tittle: Tạo Frontend Layout

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/15024

==> Attachment File:

### CODE REVIEW

**Comment by haunc-0912** [comment]
ready

File: .fuse_hidden0000000a00000001 [hunk 1]
```diff
@@ -0,0 +1,37 @@
+source "https://rubygems.org"
```
Comment (phucmh-1356) [inline .fuse_hidden0000000a00000001]: xóa file này đi em

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@import "bootstrap-sprockets";
```
Comment (phucmh-1356) [inline app/assets/stylesheets/application.scss]: trong js, scss dùng ' thay cho "

**Comment by haunc-0912** [comment]
ready

**Comment by phucmh-1356** [comment]
commented
gộp commits rồi ready lại nhé

**Comment by haunc-0912** [comment]
ready

**Pull request merged**